File: Chapter04/SizedBoxView/SizedBoxView/SizedBoxView/SizedBoxViewPage.cs
using System;
using Xamarin.Forms;

namespace SizedBoxView
{
    public class SizedBoxViewPage : ContentPage
    {
        public SizedBoxViewPage()
        {
            BackgroundColor = Color.Pink;

            Content = new BoxView
            {
                Color = Color.Navy,
                HorizontalOptions = LayoutOptions.Center,
                VerticalOptions = LayoutOptions.Center,
                WidthRequest = 200,
                HeightRequest = 100
            };
        }
    }
}

File: Chapter04/FS/SizedBoxView/SizedBoxView/SizedBoxView/SizedBoxViewPage.fs
namespace SizedBoxView

open Xamarin.Forms

type SizedBoxViewPage() = 
    inherit ContentPage(BackgroundColor = Color.Pink,
                        Content = BoxView(Color = Color.Navy,
                                          HorizontalOptions = LayoutOptions.Center,
                                          VerticalOptions = LayoutOptions.Center,
                                          WidthRequest = 200.0,
                                          HeightRequest = 100.0))

